### PR TITLE
Run 'flutter test --preview-dart-2' in strong mode

### PR DESCRIPTION
**Pull request by aam**

### CODE REVIEW

File: packages/flutter_tools/lib/src/test/flutter_platform.dart [hunk 1]
```diff
@@ -199,26 +199,51 @@ class _FlutterPlatform extends PlatformPlugin {
...
+        finalizers.add(() async {
+          printTrace('test $ourTestCount: deleting temporary bundle directory');
+          temporaryDirectory.deleteSync(recursive: true);
+        });
+
+        // copy 'vm_platform_strong.dill' into 'platform.dill'
+        final File copiedFile = fs.file(
+            artifacts.getArtifactPath(Artifact.platformKernelStrongDill))
+            .copySync(
+            tempBundleDirectory
```
Comment (a-siva) [inline packages/flutter_tools/lib/src/test/flutter_platform.dart]: Some formatting issue?

**Review by a-siva — APPROVED** [review]

File: packages/flutter_tools/lib/src/test/flutter_platform.dart (hunk 1 above)
Comment (aam) [reply packages/flutter_tools/lib/src/test/flutter_platform.dart]: Reformatted.

**Comment by Hixie** [comment]
Is there any documentation anywhere that explains the ins and outs of vm_platform_strong.dill and so forth? It would be nice to have a doc somewhere (e.g. on our wiki) that someone could read to learn how this all works, before they started trying to work on this code (since in the future, everyone contributing to the tool will be working on this code).

**Comment by aam** [comment]
> Is there any documentation anywhere that explains the ins and outs of vm_platform_strong.dill and so forth?

I'm not aware of such a doc..

Even though vm_platform_strong.dill is a temporary thing, in (hopefully) very near future, there will be only one [strong] platform.dill that is built from core 'dart:'-libraries, there will be no `--preview-dart-2` flag neither, everything will be running in dart 2.0 mode, still it has to be documented. It's just it might be worth wait a little to let things settle a bit to make sure that docs won't be outdated quickly.

**Comment by Hixie** [comment]
Yup, no rush, especially for stuff that's going to go away in a few weeks.

**Pull request merged**